src/layer/internal/predicate_iterator.rs

```diff
@@ -114,17 +114,17 @@ impl Iterator for OptInternalLayerTriplePredicateIterator {
     }
 }
 
-#[derive(Clone)]
+//#[derive(Clone)]
```

Comment:
Keep original or delete comment?

Reply:
I'm conflicted. The only reason I'm commenting this out here is cause there's a warning about dead code (it's never actually cloned). But it's totally fine to clone it if anyone decides they need to do this. This way at least it's still documented that this is intended to be clonable.